grafana/dashboards/storagegrid/fabricpool.json

```diff
@@ -0,0 +1,1951 @@
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisLabel": "",
+            "axisPlacement": "auto",
+            "barAlignment": 0,
+            "drawStyle": "line",
+            "fillOpacity": 10,
+            "gradientMode": "none",
+            "hideFrom": {
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "never",
+            "spanNulls": true,
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "bps"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 9,
+        "w": 12,
+        "x": 0,
+        "y": 21
+      },
+      "id": 39,
+      "options": {
+        "legend": {
+          "calcs": [
+            "mean",
+            "lastNotNull",
+            "max"
+          ],
+          "displayMode": "table",
+          "placement": "bottom"
+        },
+        "tooltip": {
+          "mode": "single"
+        }
+      },
+      "targets": [
+        {
+          "exemplar": false,
+          "expr": "8 * sum by (cluster,policy)(rate(storagegrid_private_load_balancer_storage_rx_bytes{cluster=~\"$SGCluster\",policy=~\"$TopPolicyRx\"}[$__interval]))",
+          "hide": false,
+          "instant": false,
+          "interval": "3m",
+          "intervalFactor": 1,
+          "legendFormat": "{{cluster}} - {{policy}} - Received",
+          "refId": "A"
+        },
+        {
+          "exemplar": false,
+          "expr": "8 * sum by (cluster,policy)(rate(storagegrid_private_load_balancer_storage_tx_bytes{cluster=~\"$SGCluster\",policy=~\"$TopPolicyTx\"}[$__interval]))",
+          "hide": false,
+          "interval": "3m",
+          "intervalFactor": 1,
+          "legendFormat": "{{cluster}} - {{policy}} - Sent",
+          "refId": "B"
+        }
+      ],
+      "title": "Top $TopResources Load Balancer Request Traffic by Policy",
+      "type": "timeseries"
+    },
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "description": "",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisLabel": "",
+            "axisPlacement": "auto",
```

Comment:
<img width="1713" alt="image" src="https://user-images.githubusercontent.com/83282894/233052456-53487030-e1c2-458d-9927-2aff61dde252.png">

These both the graphs are at policy level and then applying topk on policy. It took while to get this after few iteration of drop down. 

May be description of both panels would be helpful as topk=1 or 2 won't give 1 or 2 records but more which is based on the policy selected.